docs/reference/admin/layouts/providers.mdx

```diff
@@ -7,8 +7,13 @@ import _indexSource from '!!raw-loader!./_Example/index.tsx'
 
 # <span className="version">Interface 1.2+</span> Slots and Directives providers
 
-Before using slots and directives, you need to add providers to the application entry.
+Using slots and directives is optional. Their respective providers are included in our
```

Comment:
this is second page after overview.. I almost don't know what directives and slots are, and I get the message that it is optional. it is confusing.

Reply:
Still only those lines are relevant to scopes & directives. Keep or remove?

Reply:
you should probably move them to slots / directives chapters

Reply:
Done